src/main/java/com/dev/cinema/dao/impl/MovieDaoImpl.java

```diff
@@ -35,6 +39,7 @@ public Movie add(Movie movie) {
     }
 
     public List<Movie> getAll() {
+        log.info("Getting all movies");
```

Comment:
Good idea about commenting every attempt!
Also would be good if you`ll log if films are gotten in this function.

Reply:
@SetMyDream this is not necessary because we will get the movie list from the method